csrc/velox/CMakeLists.txt

```diff
@@ -73,16 +73,22 @@ pybind11_add_module(
 add_subdirectory(velox)
 add_subdirectory(functions)
 
+# set(CMAKE_PREFIX_PATH deps/libtorch)
+find_package(Torch REQUIRED)
```

Comment:
Another possibility that I can think of is that findTorch here is affected by some local settings of TorchArrow. I recommend to move this to the top level CMakeLists or at least very beginning of this file.